user/src/main/java/org/the_chance/honeymart/ui/feature/category/CategoriesScreen.kt

```diff
@@ -81,87 +78,84 @@ fun CategoryContent(
-                            )
-                            Text(
-                                text = state.address,
-                                style = MaterialTheme.typography.displaySmall,
-                                color = MaterialTheme.colorScheme.onSecondaryContainer,
-                                textAlign = TextAlign.Center,
-                            )
-                        }
-                        Row(
-                            modifier = Modifier.padding(vertical = MaterialTheme.dimens.space16),
-                            horizontalArrangement = Arrangement.spacedBy(MaterialTheme.dimens.space8)
-                        ) {
-                            CardChip(
-                                text = "${state.categoriesCount} Categories",
-                                icon = painterResource(id = R.drawable.boxes)
-                            )
-                            CardChip(
-                                text = "${state.productsCount} Items",
-                                icon = painterResource(id = R.drawable.box_minimalistic)
-                            )
-                        }
-                        ImageNetwork(
-                            imageUrl = state.imageUrl,
-                            contentDescription = stringResource(id = R.string.category_image),
-                            modifier = Modifier
-                                .height(198.dp)
-                                .clip(shape = RoundedCornerShape(MaterialTheme.dimens.space24))
-                                .fillMaxWidth(),
-                            contentScale = ContentScale.FillBounds
+                    }
+                    Row(
+                        modifier = Modifier.padding(vertical = MaterialTheme.dimens.space16),
+                        horizontalArrangement = Arrangement.spacedBy(MaterialTheme.dimens.space8)
+                    ) {
+                        CardChip(
+                            text = "${state.categoriesCount} Categories",
+                            icon = painterResource(id = R.drawable.boxes)
+                        )
+                        CardChip(
+                            text = "${state.productsCount} Items",
+                            icon = painterResource(id = R.drawable.box_minimalistic)
                         )
                     }
-                }
-                itemsIndexed(state.categories) { index, item ->
-                    HexagonItem(
-                        item,
-                        listener::onClickCategory,
-                        index,
-                        hexagonSize = itemSize
+                    ImageNetwork(
+                        imageUrl = "https://upload.wikimedia.org/wikipedia/commons/b/b1/Fruit_market.jpg",
+                        contentDescription = stringResource(id = R.string.category_image),
+                        modifier = Modifier
+                            .height(198.dp)
```

Comment:
You can replace this with martial spacing

Reply:
@TarekIdrees what do you mean by martial spacing?

Reply:
sorry this was typo , I mean you can use for example  Materialtheme.dimens.cardSize